ql/src/java/org/apache/hadoop/hive/ql/exec/FunctionRegistry.java

```diff
@@ -452,6 +452,7 @@ public final class FunctionRegistry {
     system.registerGenericUDF("current_schema", GenericUDFCurrentSchema.class);
     system.registerGenericUDF("current_catalog", GenericUDFCurrentCatalog.class);
     system.registerGenericUDF("current_date", GenericUDFCurrentDate.class);
+    system.registerGenericUDF("yesterday", GenericUDFYesterday.class);
```

Comment:
We need to update this file when we add a new UDF.
https://github.com/okumin/hive/commits/master/ql/src/test/results/clientpositive/llap/show_functions.q.out

Reply:
Done!